### ISSUE TITLE
Add grouping to organize different subreddits

### ISSUE THREAD

**Issue by LongJohn-Silver** [ccrama/Slide#3239]
Slide version: 6.4
Android version: 5.1.1

Because of privacy concerns I do not log in to Reddit from my smartphone, I only do this from Tor on my desktop machine, however, I like browsing subreddits without being logged in using Slide just to lurk a bit when I'm bored, and only enter my multiple accounts when I need help with something or when I totally want to engage in conversation on X or Y topic. 
Furthermore, I follow a lot of subreddits which are based around different subjects, and my home screen is a bit cluttered with so many of them. I think some form of tagging or grouping would be nice, so my home screen would have groups and within them, there would be a list of different subreddits based on specific subjects. For example, I may have a group called "Privacy" with /r/privacy, /r/privacytoolsio, /r/tor, /r/thehatedone, etc, and another one called "Gardening" with stuff like  /r/bonsai, /r/gardening, /r/mushroomgardening, /r/carnivoreplants, etc.

**Comment by ThatIsAPseudo** [comment]
In fact grouping is already available, it is called "Collections", and you can create it without logging in.
However, these collections can't be renamed, so that could be an enhancement.

**Comment by LongJohn-Silver** [comment]
@ThatIsAPseudo Yes, but in a collection all of the subreddits within it are scrambled into a simple feed, I would only like to categorize my subreddits into some sort of "folder" and then enter the one I'm interested, so that I don't have to scroll among one trillion subreddits to go to the one I'm interested in.

**Comment by ThatIsAPseudo** [comment]
Oh ok, I misunderstood your idea. Great one though !

**Comment by LongJohn-Silver** [comment]
Thanks, I hope someone could work on that, but I know it's hard, so I'll just wait until then.
